tox.ini

```diff
@@ -148,7 +148,7 @@ setenv=
   lvm_osds: CEPH_STABLE_RELEASE = luminous
 deps=
   ansible2.2: ansible==2.2.3
-  ansible2.3: ansible==2.3.1
+  ansible2.3: ansible==2.4.1
```

Comment:
Could we please add an ``ansible2.4`` factor and leave ``ansible2.3`` as-is? Thanks.

Reply:
I've made this PR to enable ``ansible2.4`` in the CI: https://github.com/ceph/ceph-build/pull/897

Reply:
yes yes, this was just a quick test to run 2.4.1, that PR is not complete.

Reply:
done